docs/api/rest_api/rest_api_reference/input/ez-types.raml

```diff
@@ -2128,401 +2140,1056 @@ View: #TODO outdated example
+    description: 'This class represents a product view input.'
     type: object
-    description: ''
+    properties:
+        ProductQuery:
+            type: object
 
 ProductView:
+    description: 'This class represents a product view.'
     type: object
-    description: ''
+    properties:
+        identifier:
+            type: string
+        ProductQuery:
+            type: object
+        Result:
+            type: object
 
 ProductVariant:
+    description: 'This class represents a product variant.'
     type: object
-    description: ''
+    properties:
+        code:
+            type: string
+        name:
+            type: string
+        Product:
+            type: Product
+        DiscriminatorAttributesList:
+            type: array
 
 ProductVariantCreate:
+    description: 'This class is used to create a product variant.'
     type: object
-    description: ''
+    properties:
+        code:
+            type: string
+        attributes:
+            type: array<int|string,mixed>
 
 ProductVariantGenerate:
+    description: 'This class is used to generate a product variant.'
     type: object
-    description: ''
+    properties:
+        attributes:
+            type: array<int|string,mixed>
 
 ProductVariantUpdate:
+    description: 'This class is used to update a product variant.'
     type: object
-    description: ''
+    properties:
+        code:
+            type: string
+        attributes:
+            type: array<int|string,mixed>
 
 ProductVariantViewInput:
+    description: 'This class represents a product variant view input.'
     type: object
-    description: ''
+    properties:
+        ProductVariantQuery:
+            type: object
 
 ProductVariantView:
+    description: 'This class represents a product variant view.'
     type: object
+    properties:
+        identifier:
+            type: string
+        ProductVariantQuery:
+            type: object
+        Result:
+            type: object
 
 RegionViewInput:
+    description: 'This class represents a region view input.'
     type: object
-    description: ''
+    properties:
+        RegionQuery:
+            type: object
 
 RegionView:
+    description: 'This class represents a region view.'
     type: object
+    properties:
+        identifier:
+            type: string
+        RegionQuery:
+            type: object
+        Result:
+            type: object
 
 ProductTypeList:
+    description: 'This class represents a product type list.'
     type: object
-    description:
+    properties:
+        ProductTypes:
+            type: ProductType[]
 
 TaxonomyEntry:
-    type: object
+    description: 'This class represents a taxonomy entry.'
+    type: BaseObject
+    properties:
+        id:
+            type: integer
+        identifier:
+            type: string
+        name:
+            type: string
+        mainLanguageCode:
+            type: string
+        names:
+            type: array
+        Parent:
+            type: Parent
```

Comment:
`No type found for name 'Parent'`

```suggestion
            type: TaxonomyEntry
```

Probably the same thing for `TaxonomyEntryCreateStruct` and `TaxonomyEntryUpdateStruct`